diff --git a/src/components/common/ErrorBoundary.tsx b/src/components/common/ErrorBoundary.tsx
index 39803d8..2e98890 100644
--- a/src/components/common/ErrorBoundary.tsx
+++ b/src/components/common/ErrorBoundary.tsx
@@ -11,2 +11,2 @@
 export function ErrorBoundary({ children }: { children: ReactNode }) {
-  const { t, messages } = useMessages();
+  const { t, labels, messages } = useMessages();
@@ -14,3 +14,3 @@
-  const fallbackRender = ({ error, resetErrorBoundary }) => {
+  const fallbackRender = ({ error }) => {
     return (
       <Column
@@ -25,7 +25,7 @@ export function ErrorBoundary({ children }: { children: ReactNode }) {
         <h1>{t(messages.error)}</h1>
         <h3>{error.message}</h3>
         <pre>{error.stack}</pre>
-        <Button onClick={resetErrorBoundary}>OK</Button>
+        <Button onClick={() => window.location.reload()}>{t(labels.refresh)}</Button>
       </Column>
     );
   };
